### ISSUE TITLE
[0.10.7] Pane highlight outline

### ISSUE THREAD

**Issue by SamVerschueren** [Microsoft/vscode#2661]
Since the insider build, it looks like the panes can be highlighted when clicked. It really looks weird to me and doesn't have added value (but maybe there is). Is this a bug or not?

![screen shot 2016-02-03 at 09 33 04](https://cloud.githubusercontent.com/assets/1913805/12776852/667e2d38-ca59-11e5-8221-1f29a63c0392.png)
![screen shot 2016-02-03 at 09 33 15](https://cloud.githubusercontent.com/assets/1913805/12776851/667ad854-ca59-11e5-827b-e6aabf5cbd21.png)

**Comment by bpasero** [comment]
@SamVerschueren this is part of our ongoing work to be more accessible: https://github.com/Microsoft/vscode/issues/2108

Indicating the currently focused element is a requirement to be accessible. You can see this in action when tabbing around the UI with the tab key.

As to wether the affordance we show is good or bad, I assign this to UX for discussion.

**Comment by bpasero** [comment]
The added value is Accessibility :]

**Comment by SamVerschueren** [comment]
Aha, makes sense :). Would it be possible to make it an option (which is default `on`) to turn this behaviour off?

**Comment by bpasero** [comment]
Yeap maybe. For now I like that everyone sees it and can report feedback on it.

**Comment by SamVerschueren** [comment]
:+1:

**Comment by SamVerschueren** [comment]
Might be more difficult. I like the highlighting when navigating with tabs. It's very clear what item is selected. But when I start debugging something, the the highlighted pane is automatically switched 3 times in a very short timespan which makes it very weird. Think we should just get the discussion going on how we can improve these things.

**Comment by bpasero** [comment]
@SamVerschueren can you capture from where to where it moves?

**Comment by SamVerschueren** [comment]
I thought I saw it when debugging, but it seems that it occurs when running my `vscode-yo` plugin.

![debug](https://cloud.githubusercontent.com/assets/1913805/12782819/fbca765c-ca7d-11e5-9f85-236bb14f615d.gif)

**Comment by bpasero** [comment]
@SamVerschueren I see, the issue is that the picker closes, focus moves back to where it was and then back into the picker.

The blue line that you see for output is not a focus indicator actually, so you might get confused there easily.

**Comment by bpasero** [comment]
It also only happens if no file is open because with a opened file, focus moves back to the file when quick picker closes.

**Comment by SamVerschueren** [comment]
> The blue line that you see for output is not a focus indicator actually, so you might get confused there easily.

Oh I see, you're right. The output panel never gets focus, although you get the idea that it receives focus because it switches so quickly.

> It also only happens if no file is open because with a opened file, focus moves back to the file when quick picker closes.

Confirmed

**Comment by bpasero** [comment]
Some things pushed and thoughts:
* closing quick open no longer restores focus to the viewlet if no editor is open
* we think about not coloring the output panel in blue but in a less strong color that does not interfere with focus
* I went over many places in the tree where we quickly removed focus from the tree which results in the large blue border and relaxed that so that it does not happen that often anymore

I am closing this one and would like to hear more feedback.

**Comment by SamVerschueren** [comment]
Thanks! Will post some feedback if it gets released.
